src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -359,4 +355,12 @@ public final PrintStream getLogger() {
 	    // does not run the construct and thus leaves the transient variables empty
 	    return System.out;
 	}
+
+	public void onAttached(Run<?, ?> run) {
```

Comment:
Plugin uses Java5, so I think that @Override cannot be used in interface. Also IntelliJ complains about it.